src/scripts/object.index.js

```diff
@@ -69,7 +69,20 @@ module.exports = (robot, res) => {
 			return;
 		}
 
-		env.searchEngine.index()
+		let commandContext = {
+			robot: robot,
+			res: res
+		};
+
+		let callback = function(callbackContext, error) {
```

Comment:
seems like you should check for if(error) and emit a message to user telling them there was a problem and they should consult logs.